datastore/ui/templates/dash.html

```diff
@@ -17,14 +17,16 @@
 
 
 <div class="container">
+
   <div class="row bg-light">
     <div class="col">
 
-      <h3>Info</h3>
+      <h1>Info</h1>
       <p>Datastore running at revision <a
           href="https://github.com/ThreeSixtyGiving/datastore/commit/{{git_rev}}">{{git_rev}}</a>.</p>
-      <p>Total Grants in the system is approximately {{total_grants|intcomma}} from {{total_datagetter_runs}} Datagetter
+      <p>Total Grants in the system is approximately {{donotcommittotal_grants|intcomma}} from {{total_datagetter_runs}} Datagetter
```

Comment:
`donotcommit` — I think maybe you didn't mean to commit this :)

Reply:
Facepalm! whoops. Thanks - have pushed a new version.